packages/trader/src/Modules/CFD/Containers/cfd-reset-password-modal.tsx

```diff
@@ -109,19 +124,21 @@ const CFDResetPasswordModal = ({
             verification_code: localStorage.getItem('cfd_reset_password_code'),
         };
 
-        WS.tradingPlatformInvestorPasswordReset(request).then(response => {
-            if (response.error && response.error.code === 'InvalidToken') {
-                renderErrorBox(response.error);
-            } else {
-                setState({
-                    ...state,
-                    is_finished: true,
-                    changed_password_type: password_type,
-                });
-                clearAddressBar();
+        WS.tradingPlatformInvestorPasswordReset(request).then(
+            (response: { error: { code: string | number; message: string }; password_type: string }) => {
```

Comment:
the type for the `error` object is the same across APIs. Can we declare it in a common place and reuse it?

Reply:
Done.